ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml

```diff
@@ -69,6 +69,27 @@ tests:
         requests:
           cpu: 100m
     workflow: aws-load-balancer-install
+- as: e2e-aws-rosa-operator
+  steps:
+    cluster_profile: aws-qe
```

Comment:
Thanks for bringing this up. Yes, we discussed this. I had to use this cluster-profile because:
- `aws` didn't work due to the missing OCM token
- dedicated cluster profile seemed like a little too much:
  - conceptionally because [the current cluster profiles](https://github.com/openshift/ci-tools/blob/fda5f7f539386fbc872cfa951e7b4af93f67808b/pkg/api/types.go#L1149-L1230) are all specific to hyperscalers
   - effort wise because it needs quite some development which is not very well justified since we are talking about an operator which gets 1 PR a month
- I thought that since ROSA SD is the primary user of AWS LB Operator it would make sense to put it under the same testing umbrella

Reply:
Great. Thanks for clarifying. Agree +1.